### ISSUE TITLE
feat(ci): support colorized output in GitHub Actions

### ISSUE THREAD

**Issue by tdabasinskas** [jlandowner/helm-chartsnap#149]
Hi,

First, thanks for a great tool!

Would it be possible to add support for colorized output in GitHub Actions? As I understand, you are using [fatih/color](https://github.com/fatih/color) module for colorized output. As per [the documentation](https://github.com/fatih/color?tab=readme-ov-file#github-actions), the module does support colorizing output in GitHub Actions, but requires explicitly setting `color.NoColor = false`. 

Could we introduce some flag or environment variable, e.g. `FORCE_COLOR`, which, when set would set `color.NoColor` to `false`?

**Comment by tdabasinskas** [comment]
Hi,

Thanks for a quick reply!

The way I understand `fatih/color` works is:

- If there's `NO_COLOR` environment variable set (regardless of its value), it explicitly disables colors.
- If there's no `NO_COLOR` nor `color.NoColor` set, it automatically decides if the colors should be used based on terminal type (which, in case of GitHub Actions results in no colors).
- If `color.NoColor` is explicitly set to `false`, it always uses colors.

So, my suggestion would be the following:

- Do not change any existing behavior of `NO_COLOR` environment variable.
- Introduce new environment variable `FORCE_COLOR` which, if set, explicitly adds `color.NoColor=false`.

This would result in no breaking changes as well as follow common patterns - some languages (e.g. Python) and frameworks already support `FORCE_COLOR` environment variable which forces the console coloring.

**Comment by jlandowner** [comment]
Thanks for the information! I learned much through this issue😳

I thought this feature should be discussed in the library and found https://github.com/fatih/color/issues/155 

By default, faith/color does not colorize outputs if stdout is not a TTY.

Unfortunately it is no merged at this point but I also found fatih's stance for env `FORCE_COLOR` . So it might be a native support in the library in the future.
https://github.com/fatih/color/pull/156#issuecomment-1387454432

But chartsnap is inspired by the existing UI snapshot testing tools like Jest.

As listed in https://force-color.org/, 
Jest supports env `FORCE_COLOR ` and also `--color` cli option to colorize outputs even if stdout is not TTY. https://jestjs.io/docs/cli#--colors

So I understand it is worth implementing this feature👍
